.github/workflows/predicators.yml

```diff
@@ -80,9 +80,50 @@ jobs:
         python-version: ${{ matrix.python-version }}
         cache: 'pip'
         cache-dependency-path: '**/setup.py'
-    - name: Run YAPF to test if python code is correctly formatted
-      uses: AlexanderMelde/yapf-action@master
+    - name: Install dependencies
+      run: |
+        pip install yapf==0.32.0
+    - name: Run yapf to detect any autoformatting changes
+      run: |
+        yapf --diff -r --style .style.yapf --exclude '**/third_party' predicators
+        yapf --diff -r --style .style.yapf scripts
+        yapf --diff -r --style .style.yapf tests
+        yapf --diff -r --style .style.yapf setup.py
+      env:
+        PYTHONHASHSEED: 0
+  isort:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
       with:
-        args: --verbose --style .style.yapf --exclude '**/third_party'
+        python-version: ${{ matrix.python-version }}
+        cache: 'pip'
+        cache-dependency-path: '**/setup.py'
     - name: Run isort to organize imports
-      uses: isort/isort-action@master
\ No newline at end of file
+      uses: isort/isort-action@master
```

Comment:
i don't think this action is configured to use the particular version we want, so we might wanna change this to run isort manually and check the diff like we do with docformatter?